tests/jballerina-unit-test/src/test/resources/test-src/types/anydata/anydata_test.bal

```diff
@@ -554,10 +554,10 @@ function testAnydataToMap() {
         convertedMCfoo = ad;
     }
 
-    map<table<map<any>>> mt = {};
+    map<table<map<anydata>>> mt = {};
     ad = mt;
     map<table<map<any>>> convertedMt;
-    if (ad is map<table<map<any>>>) {
+    if (ad is map<table<map<anydata>>>) {
```

Comment:
Shall we add some more negative test cases for type-test expression checks? 
eg.
```ballerina
table<map<any>> t = table [];
map<table<map<any>>> mt = {};
assertFalse(t is anydata);
assertFalse(mt is anydata); 
```

Reply:
Still, we are getting a compiler error for this case. 
It should be fixed via https://github.com/ballerina-platform/ballerina-lang/issues/24451